web/src/app/pages/create-survey/create-survey.component.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('CreateSurveyComponent', () => {
   let navigationServiceSpy: jasmine.SpyObj<NavigationService>;
   let route: ActivatedRouteStub;
   let activeSurvey$: Subject<Survey>;
-  let loisActiveSurvey$: Immutable.List<LocationOfInterest>;
+  let loisActiveSurvey: Immutable.List<LocationOfInterest>;
```

Comment:
Do we need to specified `ActiveSurvey` here? IIrc all data in our model is within the context of the current survey, so shouldn't be necessary.

Reply:
Make sense, I originally wanted to remove the dollar sign, but I think `lois` is good enough and it's more concise.